feature-libs/cart/added-to-cart-toast/styles/_added-to-cart-toast.scss

```diff
@@ -0,0 +1,114 @@
+:root {
+  --cx-added-to-cart-toast-width: 375px;
+  --cx-added-to-cart-toast-desktop-header-offset: 192px;
+  --cx-added-to-cart-toast-tablet-header-offset: 77px;
+  --cx-added-to-cart-toast-mobile-header-offset: 57px;
+  --cx-added-to-cart-toast-background: var(--cx-color-inverse);
+  --cx-added-to-cart-toast-translateY: -100%;
+}
+
+cx-added-to-cart-toast {
+  display: flex;
+  justify-content: flex-end;
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 1;
+  width: 100%;
+  height: 100%;
+  pointer-events: none;
+
+  .toast-container {
+    position: absolute;
+    border: 1px solid var(--cx-color-dark);
+    background: var(--cx-added-to-cart-toast-background);
+    width: 100%;
+    transition: transform 0.5s ease;
+    transform: translateY(var(--cx-added-to-cart-toast-translateY));
+    pointer-events: auto;
+
+    .added-to-cart-toast-title {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      width: 100%;
+      text-align: center;
+      font-weight: 600;
+      padding: 12px 0;
+      color: var(--cx-color-inverse);
+      background: var(--cx-color-secondary);
+    }
+
+    h3 {
```

Comment:
My opinion is to not use HTML tags for styles as in case somebody change it also CSS will need an update. I will suggest to add custom class to this tag.